textarea/elfinder-sakai/src/java/org/sakaiproject/elfinder/sakai/samigo/SamFsItem.java

```diff
@@ -0,0 +1,63 @@
+package org.sakaiproject.elfinder.sakai.samigo;
```

Comment:
Oh, that's true. I copied this from the other elfinder configurations. I can add that and update the PR. Thanks!